Sources/CLI/Volume/VolumeInspect.swift

```diff
@@ -28,8 +29,23 @@ extension Application.VolumeCommand {
         var names: [String]
 
         func run() async throws {
+            var volumes: [Volume] = []
+
             for name in names {
-                print("TODO: Inspect volume '\(name)'")
+                let response = try await ClientVolume.inspect(name)
+                volumes.append(response.volume)
+            }
+
+            let encoder = JSONEncoder()
+            encoder.outputFormatting = [.prettyPrinted, .sortedKeys]
+            encoder.dateEncodingStrategy = .iso8601
+
+            if volumes.count == 1 {
```

Comment:
Always return a list.

Otherwise scripts need to special case the output.

Did you see that we do that with any other resources?